test/libsolidity/semanticTests/storageLayoutSpecifier/last_storage_slot.sol

```diff
@@ -0,0 +1,10 @@
+contract C layout at 2**256 - 2 {
```

Comment:
This isn't really the last slot. It's only the last allowed one. We're going to want this name for #15911.

Reply:
I renamed it to "last_available_slot".